app/main.js

```diff
@@ -60,11 +60,15 @@ function displaysY (height = 600) {
 }
 
 function createTrayIcon () {
```

Comment:
Doh! I forgot that I generated the .ico file from the monochrome icon. Thank you for catching that